src/patternfly/components/InlineEdit/inline-edit.scss

```diff
@@ -0,0 +1,206 @@
+// Don't remove this magic comment. See gulpfile.js.
+// @import "../../sass-utilities/all";
+
+.pf-c-inline-edit {
+  // Group
+  --pf-c-inline-edit__group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Value group
+  --pf-c-inline-edit__group--m-value-group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Input group
+  --pf-c-inline-edit__group--m-input-group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Action group
+  --pf-c-inline-edit__group--m-action-group--item--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-inline-edit__group--m-input-group--m-action-group--item--MarginRight: 0;
+
+  // Valid action
+  --pf-c-inline-edit__action--m-valid--c-button--m-plain--Color: var(--pf-global--link--Color);
+  --pf-c-inline-edit__action--m-valid--c-button--m-plain--hover--Color: var(--pf-global--link--Color--hover);
+
+  // Icon group
+  --pf-c-inline-edit__group--m-icon-group--item--MarginRight: 0;
+
+  // Toggle
+  --pf-c-inline-edit--m-active__toggle--Color: var(--pf-global--Color--100);
+
+  // Footer group
+  --pf-c-inline-edit__group--m-footer--MarginTop: var(--pf-global--spacer--xl);
+
+  // Hidden
+  .pf-m-hidden {
+    display: none;
+    visibility: hidden;
+  }
+
+  .pf-m-display-inline-flex {
+    display: inline-flex;
+  }
+
+  // Align right
+  .pf-m-align-right {
+    margin-left: auto;
+  }
+
+  /* stylelint-disable selector-max-compound-selectors */
+  [class*="pf-m-grid"] [data-label] {
```

Comment:
is this just for the purposes of the demo? Seems like it could be problematic if we leave it in.